llvm/include/llvm/IR/DerivedTypes.h

```diff
@@ -228,6 +228,8 @@ class StructType : public Type {
     SCDB_NotContainsScalableVector = 32,
     SCDB_ContainsNonGlobalTargetExtType = 64,
     SCDB_NotContainsNonGlobalTargetExtType = 128,
+    SCDB_ContainsNonLocalTargetExtType = 64,
+    SCDB_NotContainsNonLocalTargetExtType = 128,
```

Comment:
TBH I'm not sure if it's really worth adding these flags, just for an IR verifier check. I guess it accelerates recursive checking of pathological struct types with lots of repeated element types.